### PR TITLE
MYR-1496: add new env myriad-api

### PR DESCRIPTION
**Pull request by apronny**

### CODE REVIEW

File: charts/myriad-api/templates/deployment.yaml [hunk 1]
```diff
@@ -140,6 +144,14 @@ spec:
                   name: {{ include "myriad-api.coinMarketCapSecretName" . }}
                   key: apiKey
             {{- end }}
+            {{- if .Values.config.openApiDisable }}
```
Comment (irmannmal) [inline charts/myriad-api/templates/deployment.yaml]: please move this before MYRIAD_RPC_WS_URL

**Review by irmannmal — CHANGES_REQUESTED** [review]

File: charts/myriad-api/values.yaml [hunk 2]
```diff
@@ -40,6 +40,8 @@ config:
     apiKey: ""
   coinMarketCap:
     apiKey: ""
+  openApiDisable: ""
```
Comment (irmannmal) [inline charts/myriad-api/values.yaml]: please move this before rpcWSURL

**Review by irmannmal — CHANGES_REQUESTED** [review]

File: charts/myriad-api/templates/deployment.yaml (hunk 1 above)
Comment (apronny) [reply charts/myriad-api/templates/deployment.yaml]: done

File: charts/myriad-api/values.yaml (hunk 2 above)
Comment (apronny) [reply charts/myriad-api/values.yaml]: done

**Review by irmannmal — APPROVED** [review]

**Pull request merged**